server/utils/assessments/getSections.ts

```diff
@@ -0,0 +1,14 @@
+import { ApprovedPremisesAssessment as Assessment } from '../../@types/shared'
+import Assess from '../../form-pages/assess'
+import informationSetAsNotReceived from './informationSetAsNotReceived'
+import AssessApplication from '../../form-pages/assess/assessApplication'
+
+export default (assessment: Assessment) => {
```

Comment:
If we're having these as one function per file, would it work to import and export them in an index.ts file? Or would that just cause the same dependency cycle issue?

Reply:
I haven't tried. I'll give it a go